samples/OriginalUriBaseIds.sarif

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample demonstrates the use of originalUriBaseIds."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples"
+        }
+      },
+      "originalUriBaseIds": {
+        "PROJECTROOT": {
+          "description": {
+            "text": "The root directory for all project files."

```

Comment:
>for all project files [](start = 40, length = 21)

Confusing example. most windows project files have their own directory, and therefore each project file might have its own root (it does in visual studio). You might change this to SOLUTIONROOT, of which we can more reasonably expect there to be a single definition for a scan run).

Reply:
Renamed to ENLISTMENTROOT, and re-described, per your comment below.

---
In reply to: [414790002](https://github.com/microsoft/sarif-tutorials/pull/14#discussion_r414790002) [](ancestors = 414790002)